supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,11 +2,44 @@
 [[links]]
 = Links
 
-// [[internal-links]]
-// == Internal links
+[[cross-references]]
+== Cross-references
 
-// [[external-links]]
-// == External links
+Follow these guidelines when adding cross-references within your documentation:
+
+* Include cross-references only when necessary.
+* If the information is critical, consider including it instead of cross-referencing.
+
+.Example AsciiDoc: Cross-reference
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+[[external-links]]
+== External links
+
+Follow these guidelines when linking externally:
+
+* Avoid links to external sites when possible. External links can change or be unreliable.
```

Comment:
Suggestion: Maybe consider adding `When linking to external sites, use URLs with top-level domain names, if possible.` E.g. `https://prometheus.io` vs `https://prometheus.io/docs/prometheus/latest/management_api/`. I'm not sure this is necessary, but it might help reduce broken external links.